### ISSUE TITLE
Trident versions >= RC3 can be successfully installed but don't boot

### ISSUE THREAD

**Issue by robertwolter** [trueos/trueos#273]
I use on old BIOS PC, so no UEFI is involved.

I have tried to install and boot several ISOs starting with RC3 up to Trident-18.12-PRERELEASE2-x64-20181219.iso. The installation worked fine but I couldn't boot any of them. There was an entry for Trident generated in the FreeBSD boot menu but selecting this lead to nothing else but just a blinking cursor. The boot process just hung.

I could install and boot Trident RC2 and then update/upgrade to the current version and luckily it still boots. I could also install FreeBSD-13-CURRENT and GhostBSD 18.10 and they booted without problems.

**Comment by robertwolter** [comment]
I just was told that there was no "RC3" but "prerelease" after RC2. Sorry, I forgot that.

**Comment by kmoore134** [comment]
@beanpole135 you want to transfer this issue to the Trident repo?

**Comment by beanpole135** [comment]
No, I asked him deliberately to make the issue here since it related to the bootloader from TrueOS, nothing specific to Trident.
The RC2 -> PRERELEASE transition in Project Trident was when the bootloader changed from forth to lua, so I think it is something related to that change in TrueOS/FreeBSD.

**Comment by robertwolter** [comment]
I should mention that I also installed FreeBSD-13-CURRENT and it did successfully boot. So maybe it's not a FreeBSD problem, but surely a TrueOS one.

**Comment by robertwolter** [comment]
I don't know if this helps: The first boot screen shows up  when trying to boot Trident-RELEASE-18.12, second shows successful boot of FreeBSD-13-CURRENT
![freebsdfb1](https://user-images.githubusercontent.com/18753171/51352597-5c7f7780-1aae-11e9-8951-a81ea9698f83.jpg)
![freebsdb2](https://user-images.githubusercontent.com/18753171/51352596-5c7f7780-1aae-11e9-8d52-dc262c022ddb.jpg)

**Comment by robertwolter** [comment]
I just installed TrueOS-Stable-x64-18.12.iso and it doesn't boot as well.

**Comment by robertwolter** [comment]
boot issue summary

FreeBSD*:
HDD (several installs) boots
SSD (2 installs) boots

GhostBSD18*:
HDD (several installs) boots
SSD (2 installs) boots

Trident-18.12*:
HDD (2 installs) boots
SSD (4 installs) does NOT boot

TrueOS-Stable-x64-18.12:
SSD (1 install) does NOT boot

**Comment by robertwolter** [comment]
I just tried to install from Trident-18.12-RELEASE-U3-x64-20190205.iso into a ZFS partition. Same problem as before, installs without problems but doesn’t boot.
